Error when saving document

I just got an CI error when commiting to the RDF-star CG-report : https://github.com/w3c/rdf-star/runs/4162728537?check_suite_focus=true . The detailed log is below (happened under `w3c/spec-prod@v2`):
```
    $ respec -s "http://localhost:3000/cg-spec/editors_draft.html" -o "cg-spec/editors_draft.html.built.html" --verbose -t 20 -e
      [INFO] [Timeout: 20000ms] Processing resource: http://localhost:3000/cg-spec/editors_draft.html ...
      [INFO] [Timeout: 19998ms] Launching browser
      [INFO] [Timeout: 14953ms] Navigating to http://localhost:3000/cg-spec/editors_draft.html
      [INFO] [Timeout: 12589ms] Navigation complete.
      [INFO] [Timeout: 12388ms] Using ReSpec v28.0.1
      [INFO] [Timeout: 12386ms] Processing ReSpec document...
      [INFO] [Timeout: 10615ms] Processed document.
  Error: ROR] Error when calling function .
        Plugin: core/pubsubhub
          Hint: See developer console.
  Error: ROR] undefined
      [INFO] [Timeout: 9602ms] Done.
      [FATAL] Error: Errors found during processing.
          at run (/home/runner/work/_actions/w3c/spec-prod/v2/node_modules/respec/tools/respec2html.js:241:11)
          at processTicksAndRejections (node:internal/process/task_queues:96:5)
    Failed
  Error: Process completed with exit code 1.
  Error: w3c/spec-prod/v2/action.yml (Line: 107, Col: 24):
  Error: The template is not valid. w3c/spec-prod/v2/action.yml (Line: 107, Col: 24): Error reading JToken from JsonReader. Path '', line 0, position 0.
  Error: w3c/spec-prod/v2/action.yml (Line: 117, Col: 24):
  Error: The template is not valid. w3c/spec-prod/v2/action.yml (Line: 117, Col: 24): Error reading JToken from JsonReader. Path '', line 0, position 0.
  Error: w3c/spec-prod/v2/action.yml (Line: 127, Col: 24):
  Error: The template is not valid. w3c/spec-prod/v2/action.yml (Line: 127, Col: 24): Error reading JToken from JsonReader. Path '', line 0, position 0.
  Error: w3c/spec-prod/v2/action.yml (Line: 137, Col: 21):
  Error: The template is not valid. w3c/spec-prod/v2/action.yml (Line: 137, Col: 21): Error reading JToken from JsonReader. Path '', line 0, position 0.
  Error: w3c/spec-prod/v2/action.yml (Line: 147, Col: 24):
  Error: The template is not valid. w3c/spec-prod/v2/action.yml (Line: 147, Col: 24): Error reading JToken from JsonReader. Path '', line 0, position 0.
```

--- a/src/w3c/templates/headers.js
+++ b/src/w3c/templates/headers.js
@@ -143,15 +143,15 @@ function getSpecSubTitleElem(conf) {
   return specSubTitleElem;
 }
 
-/**
- * After export, we let fixup.js handle the <details>.
- */
-sub("beforesave", doc => {
-  const details = doc.querySelector(".head details");
-  details.open = true;
-});
-
 export default (conf, options) => {
+  /**
+   * After export, we let fixup.js handle the <details>.
+   */
+  sub("beforesave", doc => {
+    const details = doc.querySelector(".head details");
+    details.open = true;
+  });
+
   return html`<div class="head">
     ${conf.logos.map(showLogo)} ${document.querySelector("h1#title")}
     ${getSpecSubTitleElem(conf)}

